error-codes.md

```diff
@@ -13,6 +13,12 @@ This document describes the problems encountered during the use of TiDB and prov
 TiDB is compatible with the error codes in MySQL, and in most cases returns the same error code as MySQL. For a list of error codes for MySQL, see [Server Error Message Reference](https://dev.mysql.com/doc/refman/5.7/en/server-error-reference.html).
 In addition, TiDB has the following unique error codes:
 
+> **Note:**
+>
+> Some error codes stand for internal errors. Normally, TiDB handles the errors rather than return to the user, so some error codes are not listed here.
```

Comment:
```suggestion
> Some error codes stand for internal errors. Normally, TiDB handles the error rather than return it to the user, so some error codes are not listed here.
```